script-based-workflows/make.R

```diff
@@ -0,0 +1,16 @@
+# This is where you set up your workflow plan,
+# a data frame with the steps of your analysis.
+
+# drake >= 7.6.2.9000 as a new interface for handling script-based plans.
```

Comment:
I think you meant "drake >= 7.6.2.9000 has".

Reply:
Ah yes, thanks